lua/pl/path.lua

```diff
@@ -398,7 +403,7 @@ function path.normcase(P)
     end
 end
 
---- normalize a path name.
+--- Get a path by normalizing a path.
```

Comment:
Just fix the case on the original, this is less clear than the original. It's just stuffing words into a word salad, not clarifying what is happening.